### DISCUSSION TITLE
moderator - number of visible videos override

### DISCUSSION THREAD

**Discussion by nullaegy** [edumeet/edumeet#846] [Ideas]
It would be nice if a moderator can override the number of videos.
At least if users leave it at 4.

**Comment by zmartinovic** [comment]
Yes, we would love this feature also, that moderator can control number of videos, not individual (authenticated) users.

**Comment by misi** [comment]
Should it be mandatory after moderator set the number of participants in focus, or it should be enabled for a normal user to override it again?

**Comment by nullaegy** [comment]
I think it would be best if the moderator can set the channel's default number, but every user can override that via their config later in their settings.
